RUFAS/biophysical/manure/storage/open_lot.py

```diff
@@ -0,0 +1,226 @@
+from copy import copy
+
+from RUFAS.biophysical.manure.manure_constants import ManureConstants
+from RUFAS.biophysical.manure.storage.solids_storage_calculator import SolidsStorageCalculator
+from RUFAS.biophysical.manure.storage.storage import Storage
+from RUFAS.biophysical.manure.storage.storage_cover import StorageCover
+from RUFAS.current_day_conditions import CurrentDayConditions
+from RUFAS.data_structures.animal_to_manure_connection import ManureStream
+from RUFAS.rufas_time import RufasTime
+from RUFAS.units import MeasurementUnits
+
+AMMONIA_EMISSION_COEFFICIENT_IN_OPEN_LOTS: float = 0.36
+"""
+Ammonia emission coefficient used for calculating nitrogen loss in an open lot (unitless).
+"""
+
+NITROUS_OXIDE_COEFFICIENT_IN_OPEN_LOTS: float = 0.02
+"""
+Nitrous oxide coefficient used for calculating nitrogen loss in an open lot (unitless).
+"""
+
+LEACHING_COEFFICIENT: float = 0.035
+"""Leaching coefficient used in the calculation of nitrogen loss in a compost bedded pack barn (unitless)."""
```

Comment:
```suggestion
"""Leaching coefficient used in the calculation of leaching N loss in an Open Lot (unitless)."""
```
This value (0.035) is the same for both CBPB and open lot, but I think they should still have separate constants in their respective files.